recipes/num2words/LICENSE

```diff
@@ -0,0 +1,145 @@
+GNU Lesser General Public License
```

Comment:
Please remove this file. The license file is in the source distribution and is called `COPYING`.